### ISSUE TITLE
Improve 'import' keyword warning

### ISSUE THREAD

**Issue by erosman** [mozilla/addons-linter#3182]
When uploading my addon I noticed a warning by the validator:
It appears that the `class` method named `import` generates the warning meant for JavaScript method `import`.

> Unsafe call to this.import for argument 0
> 
> Due to both security and performance concerns, this may not be set using dynamic values which have not been adequately sanitized. This can lead to security issues or fairly serious performance degradation.
> content/pref.js line 25 column 71

> Unsafe call to Pref.import for argument 0
> 
> Due to both security and performance concerns, this may not be set using dynamic values which have not been adequately sanitized. This can lead to security issues or fairly serious performance degradation.
> content/options.js line 722 column 67

The code in question (simplified):

```JS
class Pref {

  static importExport() {
    document.getElementById('file').addEventListener('change', (e) => this.import(e));
  }

  static import(e) {
    // code
  }

  static export() {
    // code
  }
}
```

**Comment by willdurand** [comment]
This warning comes from https://github.com/mozilla/eslint-plugin-no-unsanitized
